### PR TITLE
default to 'default' credprofile when profile not mentioned

### PR DESCRIPTION
**Pull request by samvarankashyap**
$subject
#770

### CODE REVIEW

File: linchpin/provision/roles/openshift/tasks/provision_resource_group.yml [hunk 1]
```diff
@@ -3,20 +3,16 @@
...
 - name: "set default_cred_filename when res_grp[credentials] is undefined"
   set_fact:
     cred_filename: 'openshift.yaml'
   when: res_grp['credentials'] is not defined
 
+  # sets cred profile as default if not found if not found
 - name: "set default_cred_filename when res_grp[credentials] is undefined"
   set_fact:
-    cred_profile: "{{ res_grp['credentials']['profile'] }}"
-  when: res_grp['credentials'] is defined and res_grp['credentials']['profile'] is defined
```
Comment (skatlapa) [inline linchpin/provision/roles/openshift/tasks/provision_resource_group.yml]: why did you drop profile?

File: linchpin/provision/roles/openshift/tasks/provision_resource_group.yml [hunk 2]
```diff
@@ -3,20 +3,16 @@
...
-  set_fact:
-    cred_profile: 'default'
-  when: res_grp['credentials'] is not defined
-
 - name: "set default_cred_filename when res_grp[credentials] is undefined"
   set_fact:
     cred_filename: 'openshift.yaml'
   when: res_grp['credentials'] is not defined
 
+  # sets cred profile as default if not found if not found
```
Comment (skatlapa) [inline linchpin/provision/roles/openshift/tasks/provision_resource_group.yml]: nit: double, and maybe we need to put what if not mentioned/found, what do you think?

**Comment by skatlapa** [comment]
hey, this looks good can we merge?

**Comment by samvarankashyap** [comment]
[merge]

**Pull request merged**